src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Exception {
```

Comment:
```suggestion
public class AuthenticationException extends RuntimeException  {
```

Reply:
@pkobevko Sorry, this suggestion was added by accident, in this task you should extends rom checked exception. The previous version of implementation was correct.